mne_bids/tests/test_read.py

```diff
@@ -317,6 +317,32 @@ def test_handle_coords_reading():
     with pytest.raises(RuntimeError, match='Channels do not correspond'):
         raw_test = read_raw_bids(bids_fname, bids_root)
 
+    # make sure montage is set if there are coordinates w/ 'n/a'
+    raw.info['bads'] = []
+    write_raw_bids(raw, bids_basename, bids_root, overwrite=True)
+    electrodes_dict = _from_tsv(electrodes_fname)
+    for axis in ['x', 'y', 'z']:
+        electrodes_dict[axis][0] = 'n/a'
+        electrodes_dict[axis][3] = 'n/a'
+    _to_tsv(electrodes_dict, electrodes_fname)
+
+    # set channel status as 'good' explicitly
+    channels_fname = _find_matching_sidecar(bids_fname, bids_root,
+                                            "channels.tsv",
+                                            allow_fail=True)
+    channels_dict = _from_tsv(channels_fname)
+    channels_dict['status'][0] = 'good'
+    channels_dict['status'][3] = 'good'
```

Comment:
I still don't get the point of setting the channel status here

Reply:
So my understanding is that if the channel status is `bad`, then it doesn't matter if there are electrode coordinates or not. The time where we want to default to `np.nan` is when both channel status is good and electrode coordinate is 'n/a'. For this reason, I explicitly set the channel status to 'good'  to make things explicit.

Idk how I could do this w/ `read_raw_bids` without using the private functions to alter the necessary metadata?

Reply:
humm ... `channels_dict['status']` is not used below so how is it more explicit?

Reply:
Oh yeah... ahha forgot to include `_to_tsv(channels_dict, channels_fname)`. It's updated now. 

I mean it seems that this didn't matter, since those channels were 'good' anyways, but this makes the test explicit. I could also include a version where we first set them to bad and show that montage is read in correctly as well.

Reply:
humm ... but this seems a bit unnecessary once you set `raw.info['bads'] = []`